# ダイ温度(旧ロボットベース、SCIBOT-Xのみ)GET
---
float64  value

